apps/base-docs/docs/account-abstraction-with-base-paymaster/intro-to-account-abstraction.md

```diff
@@ -0,0 +1,163 @@
+
+There are a couple options to solve for this:
+
+1. Change the protocol - hard fork! This would be a massive undertaking and most would agree that it is not a viable option.
+
+2. Change transactions, upstream (ERC-4337).
+
+In a nutshell, ERC-4337 doesn't change how Ethereum transactions work. Cryptographically signed instructions from accounts still initiate transactions to update the state of the Ethereum network.
+
+What ERC-4337 changes is everything _upstream_ of that signed transaction. It does this by introducting a new "user intent layer" that acts as a proxy for an EOA. This allows users to initiate transactions, with highly customizable smart contract wallets, but without using an EOA. Ethereum still recieves what it recieved before - signed transactions.
+
+The result is a better experience for the user without changing the Ethereum protocol.
+
+---
+
+## How Account Abstraction Works
+
+With typical Ethereum transactions an EOA initiates and signs a transaction. That transaction is sent to Ethereum's Public Mempool, is validated and added to a block, onchain.
+
+Note that the following steps are primarily happening _before_ any of that.
+
+#### 1. **Smart Contract Wallet Creation**
+
+First, a new smart contract wallet must be created for a user. This wallet is owned by it's creator who is designated as the "signer". This signer must validate any of it's future operations.
+
+As mentioned, this wallet may come with a variety of features, but it must be able to at least validate `UserOperations`s with a function called, `validateUserOp`. `validateUserOp` will check each `UserOpartion`'s signature, increment nonce, and handle the operation's fees.
+
+#### 2. **User Operation Creation**
+
+From the smart contract account, `UserOperations`s are created. Note that these are not yet transactions, but rather represent intents from the user. These intents can represent any onchain user operation.
+
+`UserOperation` includes the details of the transaction such as sender, nonce, gas limit, max fee per gas, paymaster data (if applicable), and a signature.
+
+#### 3. **Signature Generation**:
+
+The `UserOperation` is then signed using the private key associated with the initiating account. This signature serves to authenticate the transaction and validate that it was indeed initiated by the owner of the smart contract account.
+
+#### 4. **Alt Mempool**:
+
+ERC-4337 introduces an "Alt Mempool" where operations are stored until they're picked up by a "Bundler". This mempool is not very different from the transaction pool typically used in Ethereum, but note that this mempool exists earlier on in the transaction and holds user operations where Ethereum's mempool will hold signed transactions.
```

Comment:
I'd try to explain this a little deeper.  Where is this Alt mempool?  Is it in the node software like the normal transaction mempool?